ducktape/cluster/remoteaccount.py

```diff
@@ -162,6 +162,9 @@ def _get_machine_type(self):
         boot_disk = self.ssh_output(boot_disk_cmd).strip()
         disks = {}
         for d in disk_info.splitlines():
+            # ignore Warnings
```

Comment:
According to the way to get machine type (by running some command and extra cpu, memory, etc.), the fix definitely make sense to me.
I also want to mention the `_get_machine_type()` is not very robust (I know I implemented it😛). The warning message was one example (does not happen before). Hope it wouldn't happen frequently. But if it does, we might want to consider a better implementation.  @smithakoduri